test/globalize_test.rb

```diff
@@ -217,4 +217,28 @@ class GlobalizeTest < MiniTest::Spec
       end
     end
   end
+
+  describe 'with locale' do
+    def perform_with_locale(locale)
+      Thread.new do
+        Globalize.with_locale(locale) do
+          sleep 1
```

Comment:
Actually not. I tested it.
Dne 16.2.2015 1:22 "Matthieu Paret" <notifications@github.com> napsal(a):

> In test/globalize_test.rb
> <https://github.com/globalize/globalize/pull/413#discussion_r24728050>:
>
> > @@ -217,4 +217,28 @@ class GlobalizeTest < MiniTest::Spec
> >        end
> >      end
> >    end
> > +
> > +  describe 'with locale' do
> > +    def perform_with_locale(locale)
> > +      Thread.new do
> > +        Globalize.with_locale(locale) do
> > +          sleep 1
>
> 0.1 should be enough :)
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/globalize/globalize/pull/413/files#r24728050>.
>